_data/sidebars/tutorial_sidebar.yml

```
@@ -1,18 +1,27 @@
 # This is your sidebar TOC. The sidebar code loops through sections here and provides the appropriate formatting.
 
 entries:
-- title:
-  product:
-  version:
+- title: sidebar
+  product: Tutorials
+  version: v2404
```

```suggestion
  version: v202404.0
```
This would be the version of  the tutorials release, no?
This also means that we now render the master branch of the tutorials, right?
If we continue to render the develop content, this version number will not fit.

---

We do render the develop version, for reasons of easily editing.
We can also just skip the version, I don't see where it helps.
Or I can add statements which branch each subproject is rendered from, but this should already be the same as the URL of the "Edit me" button.

Would that help?

---

If we keep rendering the develop versions of the tutorials, let's simply not use the version here.

---

Removed the version altogether.

Regarding which branches to render, we now have an issue: #52